tasks/core/config.yml

```diff
@@ -49,6 +49,7 @@
     # Check if nextcloud_config is not a subset of the old preferences
```

Comment:
Remove this comment as the logic has changed now and is straightforward.

Reply:
Also move the check in the `when` directly.

Reply:
Also, let's replace the `set fact` task that sets `_nextcloud_old_preferences` with the new `_nextcloud_updated_preferences` variable.

Reply:
Alright! I think for the latter one we need both, because we also need `_nextcloud_old_preferences` for the comparison...